internal/operators/mtv/config.go

```diff
@@ -0,0 +1,16 @@
+package mtv
+
+const (
+	Name         = "mtv"
+	FullName     = "OpenShift Migration Toolkit for Virtualization"
+	Namespace    = "openshift-mtv"
+	Subscription = "mtv-operator"
+	Channel      = "release-v2.6"
```

Comment:
```suggestion

```
let it use the default channel - less maintenance for the code

Reply:
there isn't one. All that I get in OCP is `release-vX.Y`

Reply:
This one is the latest.